src/jsifier.js

```diff
@@ -151,6 +151,33 @@ function ${name}(${args}) {
 }`);
     }
 
+    if (SHARED_MEMORY) {
```

Comment:
Before this was in a check `isFunction` while now it isn't I think? That's probably fine as only functions can be proxied. Maybe worth an assertion though?

Reply:
This whole code block is now inside of `processLibraryFunction` which is only ever called on functions.